src/main/java/uk/gov/digital/ho/hocs/casework/api/CaseDataService.java

```diff
@@ -579,15 +579,18 @@ CaseSummary getCaseSummary(final UUID caseUUID) {
         auditClient.viewCaseSummaryAudit(caseData);
 
         final ActiveCaseViewData activeCaseViewData = getActiveCaseData(caseUUID);
-        final CaseSummary caseSummary = summaryBuilder
+        final CaseSummary.Builder caseSummary = summaryBuilder
                 .withPreviousCaseReference(activeCaseViewData.getPreviousCaseReference())
                 .withPreviousCaseUUID(activeCaseViewData.getPreviousCaseUUID())
-                .withPreviousCaseStageUUID(activeCaseViewData.getPreviousCaseStageUUID())
-                .build();
+                .withPreviousCaseStageUUID(activeCaseViewData.getPreviousCaseStageUUID());
+
+        if (caseDataMap.containsKey("suspended")) {
+            caseSummary.withSuspended(caseDataMap.get("suspended"));
+        }
 
         log.info("Got Case Summary for Case: {} Ref: {}", caseData.getUuid(), caseData.getReference(), value(EVENT, CASE_SUMMARY_RETRIEVED));
 
-        return caseSummary;
+        return caseSummary.build();
```

Comment:
As we're now running build() here, the log statement logs we have got the case summary before it's returned from the builder

Reply:
Altered, assigned the built CaseSummaryResponse to a var before the log, then returned that var.